src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -179,6 +186,10 @@ public void start(Map<String, String> properties) {
           partitions.add(Collections.singletonMap(JdbcSourceConnectorConstants.QUERY_NAME_KEY,
                                                   JdbcSourceConnectorConstants.QUERY_NAME_VALUE));
           break;
+        case STORED_PROCEDURE:
+          log.trace("Starting in STORED PROCEDURE mode");
+          partitions.add(Collections.singletonMap(JdbcSourceConnectorConstants.STORED_PROCEDURE,
+                                                  JdbcSourceConnectorConstants.STORED_PROCEDURE));
```

Comment:
`break;` is missed here, maybe that could be the reason of the error `"message": "Invalid value storedprocedure for configuration mode: String must be one of: , bulk, timestamp, incrementing, timestamp+incrementing"`